tianshou/data/batch.py

```diff
@@ -278,6 +278,13 @@ def get_sliced_dist(dist: TDistribution, index: IndexType) -> TDistribution:
         raise NotImplementedError(f"Unsupported distribution for slicing: {dist}")
 
 
+def get_len_of_dist(dist: Distribution) -> int:
```

Comment:
nit: why do you put dist util function under buffer?

Reply:
there's a bunch of dist utils that are scattered around collector, batch and buffer at the moment, and we don't have a dedicated module for them yet. 

I wanted to make one and also a dedicated test module in a separate PR soon (before the next release). Until then, it was just a sort of random decision where to put them